storage/src/vespa/storage/distributor/operations/external/putoperation.cpp

```diff
@@ -224,11 +224,12 @@ PutOperation::insertDatabaseEntryAndScheduleCreateBucket(
 
 void
 PutOperation::sendPutToBucketOnNode(
+        document::BucketSpace bucketSpace,
```

Comment:
Alternatively just take in a `document::Bucket` argument instead of separate space + id arguments